package.json

```diff
@@ -21,7 +21,7 @@
     "klaw": "~2.0.0",
     "markdown-it": "~8.3.1",
     "markdown-it-named-headers": "~0.0.4",
-    "marked": "~0.3.6",
+    "marked": "~0.3.9",
```

Comment:
I think we should update for ~0.3.12
https://github.com/chjj/marked/releases/tag/v0.3.12